SPECS-CROSS/cross-gcc/cross-gcc.spec

```diff
@@ -0,0 +1,161 @@
+%global debug_package %{nil}
```

Comment:
There is inconsistency in the naming of these cross compiler packages.  Why cross-gcc instead of gcc-cross?

Also I noticed that Fedora prefers a syntax like gcc-aarch64-linux so that you can see what kind of cross compiler it is.

Since we are setting the precedent here, we should decide on some uniform conventions for all of our cross compilers

Reply:
+1 for making the tuple part of the package name. If we ever expand the number of supported architectures, then we won't have to worry about conflicts. Also, this way it's clearer what architecture we're targeting with these tools.